code/modules/lighting/lighting_object.dm

```diff
@@ -113,6 +118,36 @@
 			00, 00, 00, 01
 		)
 
+	if(cr.applying_additive || cg.applying_additive || cb.applying_additive || ca.applying_additive)
+		myturf.underlays -= additive_underlay
+		additive_underlay.icon_state = "light"
+		var/arr = cr.add_r
+		var/arb = cr.add_b
+		var/arg = cr.add_g
+
+		var/agr = cg.add_r
+		var/agb = cg.add_b
+		var/agg = cg.add_g
+
+		var/abr = cb.add_r
+		var/abb = cb.add_b
+		var/abg = cb.add_g
+
+		var/aarr = ca.add_r
+		var/aarb = ca.add_b
+		var/aarg = ca.add_g
+
+		additive_underlay.color = list(
+			arr, arg, arb, 00,
+			agr, agg, agb, 00,
+			abr, abg, abb, 00,
+			aarr, aarg, aarb, 00,
+			00, 00, 00, 01
+		)
+		myturf.underlays += additive_underlay
+	else
+		myturf.underlays -= additive_underlay
```

Comment:
Can we just remove the pref and add this on top of the normal light at 50%, completely eliminating the underlay?

Reply:
Would like to see this implemented in order to test maptick and see if the maptick increase was natural causes or if the underlays are making a difference.
![image](https://github.com/BeeStation/BeeStation-Hornet/assets/26465327/42c85a87-39bb-412a-9b91-add996cee355)

Reply:
Will do